allennlp/modules/transformer/bimodal_attention.py

```diff
@@ -74,6 +112,17 @@ def _transpose_for_scores(self, x):
         x = x.view(*new_x_shape)
         return x.permute(0, 2, 1, 3)
 
+    @staticmethod
+    def _apply_mask(values: torch.FloatTensor, mask: torch.BoolTensor) -> torch.FloatTensor:
+        if len(mask.shape) == 2:
+            # We create a 4D attention mask from a 2D tensor mask.
+            # The shape is `batch_size x 1 x 1 x target_seq_len` which is broadcast
+            # to `batch_size x num_attention_heads x source_seq_len x target_seq_len`
+            mask = mask.unsqueeze(1).unsqueeze(2)
+        # `mask==1` to convert float tensors.
+        mask = (~(mask == 1)) * nn_util.min_value_of_dtype(values.dtype)
```

Comment:
I'm not sure it makes sense to use the min value for the type. For one thing, that tensor might get cast to float16 later, and for another, it might be weird to have a value there that goes to `-inf` at the drop of a hat.

Reply:
Fixed!